rune/libenclave/internal/runtime/pal/skeleton/liberpal-skeleton.c

```diff
@@ -841,18 +841,12 @@ int __pal_kill(int pid, int sig)
 
 int __pal_destroy(void)
 {
-	FILE *fp = fdopen(pal_stdio.stderr, "w");
-	if (!fp)
-		return -1;
-
```

Comment:
Need to check the validity of pal_stdio and then decide to use pal_stdio.stderr or stderr.